src/instrument.js

```diff
@@ -17,6 +17,25 @@ function hasCategory(instrument, category) {
   return false;
 }
 
+function sortByMintedAt(instruments) {
+  // sorts the instruments by minted_at property
+  // minted_at represents the time when the instrument is either minted or updated
+  instruments.sort((a, b) => {
+    return a.instrument.minted_at - b.instrument.minted_at
+  })
+  return instruments[instruments.length - 1]
```

Comment:
The method name implies it's just sorting. We should return the entire sorted array of instruments, in this case.

Reply:
fair enough! changed it